src/slider/stateful-slider-container.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import type {
+  StatefulContainerPropsT,
+  StateReducerT,
+  StateT,
+  ParamsT,
+  ChangeActionT,
+} from './types';
+
+const defaultStateReducer: StateReducerT = (type, nextState) => nextState;
+
+class StatefulSliderContainer extends React.Component<
+  StatefulContainerPropsT,
+  StateT,
+> {
+  static defaultProps = {
+    initialState: {},
+    stateReducer: defaultStateReducer,
+    onChange: () => {},
+    onMouseEnter: () => {},
+    onMouseLeave: () => {},
+    onFocus: () => {},
+    onBlur: () => {},
+  };
+
+  state = {...this.props.initialState};
+
+  onChange = (e: SyntheticInputEvent<HTMLInputElement>, params: ParamsT) => {
```

Comment:
Change to the `{}` type arg for this handler

Reply:
ok